### PR TITLE
Improve wording of missing page error message

### PR DESCRIPTION
**Pull request by CleanMachine1**
I am surprised that this issue has gone unnoticed, it must be because we have so pages documented ;) 

Also I don't know whether you agree with the de-capitalization, however I feel this is correct since with using the word `PRs`, you are indicating that they are a part of an acronym

### CODE REVIEW

**Comment by marchersimon** [comment]
How about we use the same message as `tldr-node-client`: 
>If you want to contribute it, feel free to send a pull request to: https://www.github.com/tldr-pages/tldr.

**Comment by CleanMachine1** [comment]
Sure, I don't see a problem with that.

File: tldr.py [hunk 1]
```diff
@@ -468,7 +468,7 @@ def main():
             if not result:
                 sys.exit((
                     "`{cmd}` documentation is not available. "
-                    "Consider contributing Pull Request to "
+                    "If you want to contribute it, feel free to send a pull request to:"
```
Comment (CleanMachine1) [inline tldr.py:471]: I might need to add a new line like the rest

Comment (MasterOdin) [reply tldr.py:471]: Yeah, add a newline after just the first line.

Comment (CleanMachine1) [reply tldr.py:471]: Will do!

Comment (CleanMachine1) [reply tldr.py]: https://github.com/tldr-pages/tldr-python-client/pull/163/commits/654e7600d4110705c9c2ab43ce1a80caeaf698c1

**Comment by CleanMachine1** [comment]
@MasterOdin I don't see the problem here. Is it setup to run with 3 lines instead of 4, or have I made a mistake?

File: tldr.py [hunk 2]
```diff
@@ -468,7 +468,8 @@ def main():
             if not result:
                 sys.exit((
                     "`{cmd}` documentation is not available. "
-                    "Consider contributing Pull Request to "
+                    "If you want to contribute it, "
+                    "feel free to send a pull request to:"
```
Comment (CleanMachine1) [inline tldr.py:472]:
```suggestion
                    "If you want to contribute it, 
                    feel free to send a pull request to:"
```

```suggestion
                    "If you want to contribute it, \nfeel free to send a pull request to:"
```

**Comment by CleanMachine1** [comment]
@MasterOdin You know the backend to this, I am sure you will look at this and find a fix within 20 seconds, because you know how this works.

Sorry for creating 9 waste commits, we'll squash when merging.

File: tests/test_tldr.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def test_error_message():
     with mock.patch("sys.argv", ["tldr", "73eb6f19cd6f"]):
         with pytest.raises(SystemExit) as pytest_wrapped_e:
             tldr.main()
-        correct_output = "`73eb6f19cd6f` documentation is not available. If you want to contribute it, feel free to send a pull request to: https://github.com/tldr-pages/tldr." # noqa
+        correct_output = "`73eb6f19cd6f` documentation is not available. If you want to contribute it, feel free to send a pull request to: https://github.com/tldr-pages/tldr" # noqa
```
Comment (marchersimon) [inline tests/test_tldr.py:30]: Not the period 😢

Comment (CleanMachine1) [reply tests/test_tldr.py:30]: It will probably be readded, I was just testing. I can't find the issue which is breaking the test.

**Comment by MasterOdin** [comment]
@CleanMachine1 No problem! I made a commit that resolves the test failures. Don't worry about squashing on your end, I will handle doing that when I go to merge the PR.

**Comment by CleanMachine1** [comment]
Ok, well, I'll leave this in your hands in terms of merging.

But I have 1 question, how long will this simple change take till this in the Arch Repos'

**Comment by MasterOdin** [comment]
> But I have 1 question, how long will this simple change take till this in the Arch Repos'

I'm not sure as I have no hand in maintaining it. In the past it's been decently quick to update.

**Comment by CleanMachine1** [comment]
Normally at a tag I think at least

**Pull request merged**